### DISCUSSION TITLE
Github not accepting airmail.cc email domain.

### DISCUSSION THREAD

**Discussion by dowdt** [community/community#116595] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I tried adding an airmail.cc domain to my account's emails and got the following error: 
`Error adding <myemail>@airmail.cc: Email domain could not be verified`

I've been using this email in many repos. Is there anything I can do to verify this email?

**Comment by mruncreative** [comment]
Other valid domains are not being accepted either. You shouldn't constrain email domains in the first place. It's against the spirit of email and the interests of everyone who isn't a megacorp.
